### ISSUE TITLE
Invalid path for /etc and /root

### ISSUE THREAD

**Issue by dvanders** [jotta/jotta-cli-issues#11]
**jotta-cli release (`jotta-cli version`):**

```jotta-cli     version 0.3.4269```

**Description of problem:**

```
# jotta-cli add /etc/
Error: [2] Invalid path
# jotta-cli add /root/
Error: [2] Invalid path
```

It's understandable to exclude /usr, full of binaries, but /etc/ and /root/ are pretty important to back up.
Also /var should be allowable, pretty please?

**Comment by Kimbsen** [comment]
We did not intend to block those paths. We will allow them in the next update.

**Comment by Kimbsen** [comment]
This should be resolved now with the 0.4 release.
